mcs/build/profiles/build.make

```diff
@@ -147,8 +147,8 @@ export VBCS_LOCATION
 
 start-compiler-server:
 	echo Attempting to start compiler server...
-	./build/start-compiler-server.sh '$(realpath $(topdir))' '$(realpath $(topdir)/build)/compiler-server.log' '$(COMPILER_SERVER_PIPENAME)'
+	$(topdir)/build/start-compiler-server.sh '$(realpath $(topdir))' '$(realpath $(topdir)/build)/compiler-server.log' '$(COMPILER_SERVER_PIPENAME)'
```

Comment:
Isn't this reverting #14117

Reply:
this might be a bad line merge, i'll check. thanks.

Reply:
What is the status on this?

Reply:
I genuinely don't know. I think topdir is correct and the bitrot was in the realpath part